apis/votingApiCall.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+const addVote = (article_id, flag) => {
```

Comment:
- Good to see the axios requests extracted into their own folder - nice and reusable

Reply:
Just thinking of another dev - could rename flag to `addUpVote` - then its either true or false as you pass in